packages/@webex/plugin-cc/src/services/config/types.ts

```diff
@@ -113,6 +113,13 @@ export type AgentLogin = {
   loginOption: LoginOption;
 };
 
+export interface StateChange {
```

Comment:
it should be type.

Reply:
Any reason we can't use an interface here?

Reply:
We generally use type where it contains just one type or a combination of multiple interfaces and types.

Reply:
we use type instead of interface